stable/prometheus-adapter/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: prometheus-adapter
-version: 1.4.0
+version: 1.4.1
```

Comment:
This should be a major version increase since it will break for people on older releases

Reply:
Still not major increase

Reply:
oh, major?! sorry, will fix